web/themes/custom/rebartik/src/scss/pages/home/main_block.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../_global';
+@import '_mixin';
```

Comment:
Why do you plan to use the '_mixin.scss' separatelly from '_global.scss'?

Generally, we have to place mixins next to variables and include them in global.